core/mgmt/v1beta/mgmt_public_service.proto

```diff
@@ -64,6 +64,28 @@ service MgmtPublicService {
     option (google.api.method_visibility).restriction = "INTERNAL";
   }
 
+  // Get the authenticated user
+  //
+  // Accesses the authenticated user.
+  rpc GetAuthenticatedUser(GetAuthenticatedUserRequest) returns (GetAuthenticatedUserResponse) {
+    option (google.api.http) = {get: "/v1beta/user"};
+    option (google.api.method_signature) = "user,update_mask";
+  }
+
+  // Update the authenticated user
+  //
+  // Accesses and updates the authenticated user.
```

Comment:
```suggestion
  // Updates the information of the authenticated user.
```

I think the "access" bit made sense only as "accessed by ID". "Authenticated user" already reflects how the user is found.